src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java

```diff
@@ -132,6 +145,16 @@ public void deleteRoles(final Node node) throws RepositoryException {
                 rdfTypes.addAll(x.getTypes());
             });
 
+        // If we get the default Authorization and it contains acl:accessTo properties,
```

Comment:
It is unclear to me what you are achieving with this block... particularly since the comment is somewhat misleading by referring to the "default Authorization", which is not found until line:169 further down.